src/Tools/Source/CompilerGeneratorTools/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# Project-specific settings:
+[*.{cs,vb}]
+# Local functions are camelCase
+dotnet_naming_style.local_function_style.capitalization = camel_case
+
+# CSharp code style settings:
+[*.cs]
```

Comment:
Not blocking this PR: Instead of duplicating config files, I wonder if we should consolidate them and use `[Compiler/*.cs, Tools/*.cs, ...]` formats to re-use across compiler directories.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/32962](https://github.com/dotnet/roslyn/issues/32962) to follow up on this.

---
In reply to: [252058042](https://github.com/dotnet/roslyn/pull/32899#discussion_r252058042) [](ancestors = 252058042)